Newhomepage

Snekkerbuksa opened the pull request:
See push comments

Snekkerbuksa committed:
Fixed errors

Snekkerbuksa committed:
Removed extra css and reverted styles.css

Snekkerbuksa committed:
pause

Snekkerbuksa committed:
Small fix

- Added home.less
- Fixed footer
- Fixed material icons

Snekkerbuksa committed:
Oops

- Deleted img/css files

Snekkerbuksa committed:
Group page cut #1

Snekkerbuksa committed:
merged

Snekkerbuksa committed:
Animated see more list

Snekkerbuksa committed:
Polish

Snekkerbuksa committed:
So far

This is a tentative first draft of what the finished group page might look like. I've only done java script for "what else are we reading" - all the other icons need it. Also, I may have used a few dirty tricks to get what I want, but I'm sure you can spot them easily and add more professional flare.

Snekkerbuksa committed:
Ops. Tiny adjustment

Just forgot to change the color of "Title of the book" in reading tab panel

jscasca commented:
[anon_group.pdf](https://github.com/jscasca/prlg_styles/files/2907613/anon_group.pdf)

Should we go with container fluid? It feels a bit weird from me, as things change size but their position in the page stay the same.
Not sure why the icons are not appearing in the pdf but ignore that.
We would have to rework how it displays on tablets and phones.

Also for the style doc, I was thinking just having one single doc so styles can import groups and not call both stylesheets in the html doc

Snekkerbuksa committed:
Updates

Changed background
Added dropdown menu to profile button in nav bar

Snekkerbuksa commented:
Hey.

Hmm. All I want is for the elements inside the profile box change equally (maximise/minimise depending on the screen size of device used) i.e. keep their ratio and place. Do you know a better way to do this? 

Can you explain your last sentence to me after class tonight - I’m not grasping it.

Sc.

> On 27 Feb 2019, at 7:37 am, jscasca <notifications@github.com> wrote:
> 
> anon_group.pdf <https://github.com/jscasca/prlg_styles/files/2907613/anon_group.pdf>
> Should we got with container fluid? It feels a bit weird from me, as things change size but their position in the page stay the same.
> Not sure why the icon are not appearing in the pdf but ignore that.
> We would have to rework how it appears on tablets and phone.
> 
> Also for the style doc, I was thinking just having one single doc so styles can import groups and not call both stylesheets in the html doc
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/jscasca/prlg_styles/pull/2#issuecomment-467624037>, or mute the thread <https://github.com/notifications/unsubscribe-auth/Ar9907MVjXQWytnlbr0piMXp32m_4wdoks5vRakLgaJpZM4bGZ_s>.
>

Snekkerbuksa commented:
Oh, I think I got it. I’ll keep the group.less only and have it draw from styles.

Anyway, I have made quite a few changes and I’ll send through another pull soon.


> On 27 Feb 2019, at 7:37 am, jscasca <notifications@github.com> wrote:
> 
> anon_group.pdf <https://github.com/jscasca/prlg_styles/files/2907613/anon_group.pdf>
> Should we got with container fluid? It feels a bit weird from me, as things change size but their position in the page stay the same.
> Not sure why the icon are not appearing in the pdf but ignore that.
> We would have to rework how it appears on tablets and phone.
> 
> Also for the style doc, I was thinking just having one single doc so styles can import groups and not call both stylesheets in the html doc
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/jscasca/prlg_styles/pull/2#issuecomment-467624037>, or mute the thread <https://github.com/notifications/unsubscribe-auth/Ar9907MVjXQWytnlbr0piMXp32m_4wdoks5vRakLgaJpZM4bGZ_s>.
>

Snekkerbuksa committed:
Saving before doing profile picture

Snekkerbuksa committed:
Profile image section

Added edit profile image section
Made members placeholder images black

Snekkerbuksa committed:
Update group.less

Snekkerbuksa committed:
Admin page coming along

Contains:
- "Change image" btn with Java
- "Edit" club description with Java
- A right hand panel designed for admin only with "more" btns (that I cannot make listen to CSS, please help)
